### PR TITLE
Map entity methods, params, etc

### PR DESCRIPTION
**Pull request by apple502j**
Mostly 21w13a stuff but some old ones are included too.

### CODE REVIEW

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 1]
```diff
@@ -157,6 +165,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 		ARG 2 damageMultiplier
 	METHOD method_23667 swingHand (Lnet/minecraft/class_1268;Z)V
 		ARG 1 hand
+		ARG 2 notifyMyself
```
Comment (liach) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: Hmm, might need a better name

File: mappings/net/minecraft/entity/passive/MooshroomEntity.mapping [hunk 2]
```diff
@@ -3,13 +3,16 @@ CLASS net/minecraft/class_1438 net/minecraft/entity/passive/MooshroomEntity
 	FIELD field_18106 stewEffect Lnet/minecraft/class_1291;
 	FIELD field_18107 stewEffectDuration I
 	FIELD field_18108 lightningId Ljava/util/UUID;
+	FIELD field_30339 DIFFERENT_TYPE_CHANCE I
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/MooshroomEntity.mapping]:
```suggestion
	FIELD field_30339 MUTATION_CHANCE I
```

File: mappings/net/minecraft/entity/Entity.mapping [hunk 3]
```diff
@@ -243,8 +247,13 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT
 		COMMENT @implNote The swim height of 0 allows short entities like baby animals
 		COMMENT to start swimming to avoid suffocation.
+	METHOD method_29494 (Lnet/minecraft/class_2680;)Z
+		ARG 0 block
```
Comment (enbrain) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		ARG 0 state
```

File: mappings/net/minecraft/entity/Entity.mapping [hunk 4]
```diff
@@ -243,8 +247,13 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
 	METHOD method_29495 refreshPositionAfterTeleport (Lnet/minecraft/class_243;)V
 		ARG 1 pos
+	METHOD method_30022 (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Z
+		ARG 1 block
```
Comment (enbrain) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		ARG 1 state
```

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 5]
```diff
@@ -174,17 +183,22 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 		COMMENT
 		COMMENT <p>This checks both the entity's main and off hand.
 		ARG 1 item
+	METHOD method_24519 (Lnet/minecraft/class_1792;Lnet/minecraft/class_1799;)Z
+		ARG 1 item
```
Comment (enbrain) [inline mappings/net/minecraft/entity/LivingEntity.mapping]:
```suggestion
		ARG 1 stack
```

**Review by YanisBft — APPROVED** [review]
Seems good besides liach's comments

File: mappings/net/minecraft/entity/ExperienceOrbEntity.mapping [hunk 6]
```diff
@@ -26,6 +27,18 @@ CLASS net/minecraft/class_1303 net/minecraft/entity/ExperienceOrbEntity
...
 	METHOD method_31497 merge (Lnet/minecraft/class_1303;)V
 		ARG 1 other
+	METHOD method_31498 onEverySecond ()V
+		COMMENT Called every second (every 20 ticks).
+		COMMENT
+		COMMENT @implSpec This method first checks if the orb still has a nearby {@link #target},
+		COMMENT and assigns a new target if there is none. It then tries to merge nearby experience orbs.
+	METHOD method_35051 repairRecursively (Lnet/minecraft/class_1657;I)I
+		COMMENT Repairs a player's gears using the experience recursively, until the experience is
+		COMMENT all used or all gears are repaired.
```
Comment (enbrain) [inline mappings/net/minecraft/entity/ExperienceOrbEntity.mapping]:
```suggestion
	METHOD method_35051 repairPlayerGears (Lnet/minecraft/class_1657;I)I
		COMMENT Repairs a player's gears using the experience recursively, until the experience is
		COMMENT all used or all gears are repaired.
```
IMO `Recursively` isn't necessary unless there is a non-recursive version.

File: mappings/net/minecraft/entity/passive/AxolotlEntity.mapping [hunk 7]
```diff
@@ -19,6 +26,9 @@ CLASS net/minecraft/class_5762 net/minecraft/entity/passive/AxolotlEntity
 		ARG 1 air
 	METHOD method_33231 setPlayingDead (Z)V
 		ARG 1 playingDead
+	METHOD method_35175 giveBuffToAttacker (Lnet/minecraft/class_5762;)V
```
Comment (enbrain) [inline mappings/net/minecraft/entity/passive/AxolotlEntity.mapping]:
```suggestion
	METHOD method_35175 buffAttacker (Lnet/minecraft/class_5762;)V
```

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: This param indicates that the swing action should be sent to client. Usually if the client initiates a swing hand, it animates first and sends a packet (you can see what happens when you click when your network is laggy). This is `true` when the hand swing is initiated by the server. So:
```suggestion
		ARG 2 fromServer
```
(can be potentially improved, feel free to suggest more descriptive names)

Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Or maybe 
```suggestion
		ARG 2 fromServerPlayer
```
to explicitly indicate, as this seems only be called from ender eye item's code

**Review by liach — APPROVED** [review]

**Pull request merged**